packages/components/select/src/useOption.ts

```diff
@@ -77,8 +77,8 @@ export function useOption(props, states) {
     () => props.value,
     (val, oldVal) => {
       const { remote, valueKey } = select.props
-
-      if (val !== oldVal) {
+      const shouldUpdate = remote ? val !== oldVal : !isEqual(val, oldVal)
+      if (shouldUpdate) {
```

Comment:
I dont have the whole context of but it we pass remote to `<el-select>` the issue still exist and im asking why do we handle remote differently then a non-remote. I think it should be the same all the time ?
```suggestion

      if (!isEqual(val, oldVal)) {
```

Reply:
Yes, It looks like unreasonable now, I will try to solve the previous problem. But I hope to deal with it this way for the time being, and replace it if someone finds a better solution

Reply:
Yes, It looks like unreasonable now, I will try to solve the previous problem. But I hope to deal with it this way for the time being, and replace it if someone finds a better solution